_data/themes/beach.yml

```diff
@@ -0,0 +1,93 @@
+# Font configurations
+# 'sans' is applied to navigation and body text
+# 'serif' is applied to headings
+# 'normal' defines the default font weight
+# 'bold' defines the bold font weight
+fonts:
+  sans: >
+    "Source Sans Pro", "Helvetica Neue", "Helvetica", "Roboto", "Arial", sans-serif
+  serif: >
+    "Baskerville", "Georgia", "Cambria", "Times New Roman", "Times", serif
+  normal: 300
+  bold: 500
+
+# Color configurations
+colors:
+# primary changes content link color, primary button bg color, return to top link color
+  primary: 'DodgerBlue'
+# primary darker changes content link hover color, primary button hover bg color, return to top link hover color
+  primary-darker: 'pink'
+
+# header site title, active top nav link color, left nav link color, link color in primary footer and main footer
+# text in usa-banner ???
+  base: 'DarkSlateGray'
+
+# Link colors
+  visited: 'Purple'
+
+# USA banner colors
+  usa-banner:
+    usa-banner-bg: 'LightBlue'
+    usa-banner-text: 'Black'
+    usa-banner-link: 'Gray'
+    usa-banner-link-hover: 'Gray'
+
+# Alt Section bg color
+  alt-section:
+    bg-color: 'Cornsilk'
+    header-color: 'Blue'
+    text-color: 'Black'
+    link-color: 'red'
+    link-hover-color: 'orange'
+
+# Hero colors
+  hero:
+    hero-bg: 'white'
+    hero-header: 'DarkSlateGray'
+    hero-header-alt: 'DarkSlateGray'
+    hero-text: 'Gray'
+    hero-link: 'DodgerBlue'
+    hero-button-bg: 'DodgerBlue'
+    hero-button-text: 'white'
+
+# Navigation colors
+# Top nav colors
+  top-navigation:
+    top-nav-bg: 'DarkSlateGray'
+    top-nav-link: 'white'
+    top-nav-link-hover: 'green'
+    top-nav-link-current: 'white'
+    top-nav-dropdown-bg: '#6eb7b7'
+    top-nav-dropdown-link-hover-bg: 'DarkSlateGray'
+# Side nav colors
+  side-navigation:
+    side-nav-bg: 'white'
+    side-nav-link: 'black'
+    side-nav-link-hover: 'white'
+    side-nav-link-hover-bg: 'DarkSlateGray'
+    side-nav-link-current: 'DarkSlateGray'
+
+# Footer colors
+  footer:
+    primary-footer-bg: 'Gainsboro'
+    primary-footer-link: 'black'
+    footer-bg: 'DarkSlateGray'
+    footer-text: 'white'
+    footer-link: 'LightBlue'
+
+
+###
+###
+###
+# dont see any changes what does these change, are they needed?
+  secondary-light: 'lime'
```

Comment:
moved all these to the bottom as they were not changing anything - maybe remove?

Reply:
Yeah I think if they don't have an impact on the overall theme we should cut them.